deploys/helm/secor/config/secor.kubernetes-prod.properties

```diff
@@ -181,7 +193,7 @@ kafka.dual.commit.enabled=false
 
 # Storage offset.
 # Possible values: "zookeeper" to read offset from zookeeper or "kafka" to read offset from kafka consumer topic
-kafka.offsets.storage=kafka
+kafka.offsets.storage=zookeeper
```

Comment:
Is the storage switch just for your use case or do you think by default all k8 installations should use ZK storage?

Reply:
@HenryCaiHaiying :

In newer kafka libraries with the new consumer code, it by default stores offsets in kafka itself I think.

Default before in this project has been what's defined in the `src/main/config/secor.common.properties` which is `zookeeper`.

I did try it with `kafka` a bit, but returned back to using `zookeeper` as this is what the `com.pinterest.secor.main.ProgressMonitorMain` uses for monitoring offsets.. 

So until we write to new consumer library code, I think it might be better to stick with `zookeeper` or hear what others have been testing this with.

So this is my reasining for at least running it with zk storage ourselves.